src/makeEslinter.ts

```diff
@@ -0,0 +1,50 @@
+import * as eslinttypes from 'eslint'; // import for types alone
+import { NormalizedMessage } from './NormalizedMessage';
+import { FsHelper } from './FsHelper';
+import { makeCreateNormalizedMessageFromEsLintFailure } from './NormalizedMessageFactories';
+
+export function makeEslinter(eslintOptions: object) {
```

Comment:
maybe we could call it `createEslinter` as it's a more common name for factory function in the javascript world :) I see that we use `make` word for a factory of factory function to not double the `create` word :)

Reply:
Happy to switch it 😄